docs/technical/continuous-deployment-applications.md

```diff
@@ -0,0 +1,33 @@
+# Continuous Deployment Strategy for Applications
+
+This document outlines a high-level strategy and approach for Continuous Deployment pipelines across the whole Bürokratt system.
+
+Continuous Deployment (CD) is the practice of automatically releasing changes from the (`main`) branch frequently and in doing so "continuously" ensuring that the code "integrates" well with other code.
+
+By combining Continuous Integration and infrastructure as code (IaC), we're able to achieve identical deployments with the confidence to be able to deploy to production at any time. CD allows us to automate the entire process from code commit to production if CI/CD tests and security checks pass.
```

Comment:
It's very minor but this talks about CI, whereas the paragraph above talks about CD. I think it would be worth aligning the two paragraphs